pandas-stubs/core/frame.pyi

```diff
@@ -1440,9 +1441,9 @@ class DataFrame(NDFrame, OpsMixin):
     ) -> DataFrame: ...
     def astype(
         self,
-        dtype: _str | Dtype | Mapping[HashableT, _str | Dtype] | Series,
+        dtype: AstypeArg | dict[Any, AstypeArg],
```

Comment:
Change to `AstypeArg | Mapping[Hashable, AstypeArg] | Series`

Reply:
Done sir